lumigator/python/mzai/backend/backend/services/jobs.py

```diff
@@ -24,6 +26,27 @@
 
 
 class JobService:
+    # set storage path
+    storage_path = f"s3://{ Path(settings.S3_BUCKET) / settings.S3_JOB_RESULTS_PREFIX }/"
+
+    # TODO: we can set a different amount of GPUs per type of job
```

Comment:
❓ should this `TODO` be removed?

Reply:
I'd keep this up until we make a decision whether we want to set different amount of GPUs per type of job